Plots/MapProjections/NCL_proj_2_lg.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
Please revise and format the docstrings accordingly as in the other functions

Reply:
Went ahead and based revisions on what we have on the geocat-examples page and have uploaded new version

Reply:
Good!